.gitignore

```diff
@@ -14,6 +14,7 @@
 *.log
 *.pot
 *.pyc
+.envrc
```

Comment:
Could you add such files to your global gitignore instead? Otherwise we'll end up with a list covering all existing tools and editors. 😄

Reply:
Ok sorry :)